### PR TITLE
Replaced Bootstrap Tour with IntroJs in Stopwatch activity

### PR DESCRIPTION
**Pull request by saakxshi**
I have migrated Stopwatch activity tutorial from bootstrap-tour to IntroJs
Issue: https://github.com/llaske/sugarizer/issues/1090

[screen-capture (3).webm](https://user-images.githubusercontent.com/91880528/214114912-61e33bf3-6ff7-4d80-8c09-347f5a8bee03.webm)

### CODE REVIEW

**Comment by llaske** [comment]
Thanks @saakxshi .
Two things:
- Popup title has not the right color: should be white not gray.
- Avoid reformatting files (TAB/Space). Your update should be limited to updated lines. Guess it's an option in your IDE.

**Comment by llaske** [comment]
> Okay, I'll take care of auto-formatting for the upcoming activities. I've disabled auto-formatting, which automatically formatted my files on saving.

Please update this PR without auto-formatting else it's difficult to see your changes in updated files.

**Comment by saakxshi** [comment]
> Please update this PR without auto-formatting else it's difficult to see your changes in updated files.

I've made the changes , lemme know if it's correct.

File: activities/Stopwatch.activity/index.html [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
-    <link rel="stylesheet" href="css/activity.css">
-    <script>if (typeof module === 'object') {window.module = module; module = undefined;}</script>
-    <script src="lib/jquery.min.js"></script>
-    <script src="lib/bootstrap.min.js"></script>
-    <script src="lib/bootstrap-tour.min.js"></script>
-    <script>if (window.module) module = window.module;</script>
-    <script data-main="js/loader" src="lib/require.js"></script>
-    <script type="text/javascript" src="../../cordova.js"></script>
-  </head>
+<head>
```
Comment (llaske) [inline activities/Stopwatch.activity/index.html:4]: Do not update indentation/spacing.
Only updated lines should be changed in files.

Comment (saakxshi) [reply activities/Stopwatch.activity/index.html]: Fixed.

**Pull request closed without merging**